cross-chain/arbitrum/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can we align that file to the root `solidity` package https://github.com/keep-network/tbtc-v2/blob/main/solidity/tsconfig.json ?